test/basicShapePolygonTest.js

```diff
@@ -4,29 +4,31 @@
 (function () {
   suite('offsetPath', function () {
     test('polygon', function () {
+      // FIXME: Create a helper test function to handle the testing of interpolating when using percentages.
       var offsetPathParse = internalScope.offsetPathParse;
       var assertTransformInterpolation = internalScope.assertTransformInterpolation;
 
       var result = offsetPathParse('polygon(0px 0px, 150px 200px, 250px 150px, 200px -150px)');
```

Comment:
Nit: Instead of this result variable you can split the function call parameters on separate lines.